src/components/MainListInfoForm/MainListInfoForm.tsx

```diff
@@ -84,13 +87,14 @@ export const MainListInfoForm = (
           key={recordTypeOptions.length}
           className={css.recordTypeField}
         >
-          {/* @ts-ignore:next-line */}
-          <Select
+          <Selection
             required
             value={value}
             name={FIELD_NAMES.RECORD_TYPE}
             dataOptions={recordTypeOptions}
-            onChange={onChangeHandler}
+            onChange={(value: string) => {
+              onSelectHandler(FIELD_NAMES.RECORD_TYPE, value)
```

Comment:
If you're already making a function `onSelectHandler` above, why not put all the logic there (so the prop can be just `onChange={onSelectHandler}`?

Reply:
onChange of selection works differently from other components. It passes a string value instead of an event object, so we need an adapter wrapper.